### DISCUSSION TITLE
Entrification: Global sets and categories

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#14650] [General]
Hey all,

Just out of curiosity, I might have missed a memo, but what is the state of the 'Entrification' plan? I've read in a blog a while ago that it would not be possible in Craft 5 to add new global sets or categories and would be completely gone in Craft 6. However, in the latest beta of v5 this is still possible.

IMHO this works just fine, especially for a global set containing site wide settings.

But I would really like to now where this is heading, so I can make solid choices on how to setup my Craft structure.

Thanks!

Cheers

**Comment by thupsi** [comment]
I think the latest update is that the "Entrification" will be completed in Craft 6. In the meantime you can still create and use Categories, Globals and Tags in Craft 5.

As of now, the only benefit of keeping those types of elements is that they live on their own place in the global sidebar, which helps organizing things for your authors. I've made a [FR](https://github.com/craftcms/cms/discussions/14609) to enable the same kind of organization for arbitrary section/source groups. If/when that goes through I think it would be best for you to just go ahead and entrify everything.

**Reply by dolphiq-allard** [reply]
Yes, I've read your FR after posting mine. I think your suggestion is a good one. It would be great to have them in their own place in the sidebar. Plus, what would be really helpful is to be able to store singels in those sections, and not have them available only in the 'Singles' section, which kind of is a weird place for a set of global settings.

Thanks for your reply

**Reply by thupsi** [reply]
I 100% agree about singles. I think each single should be an individual "source" in the entries view and should be able to get "grouped" with other sections or sources.

**Comment by brandonkelly** [comment]
Craft 5.9.0 is out with the ability to create custom entry index pages (#17779) 🎉
